mibs/cdata/SNMPv2-SMI

```diff
@@ -0,0 +1,119 @@
+SNMPv2-SMI DEFINITIONS ::= BEGIN

```

Comment:
Please remove any standard mib file (that is already in ./mibs/) from this directory. Only vendor specific mibs should be here.
If a standard mib is missing from ./mibs/, then please add it there instead od the vendor specific directory